test/Reachability/MavenSpec.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Reachability.MavenSpec (spec) where
+
+import App.Fossa.Reachability.Maven (getJarsByBuild, isValidJar)
+import Control.Effect.Lift (sendIO)
+import Path (Abs, Dir, File, Path, mkRelDir, mkRelFile, (</>))
+import Path.IO qualified as PIO
+import Test.Effect (it', shouldBe')
+import Test.Hspec (Spec, describe)
+
+spec :: Spec
+spec = describe "Maven" $ do
+  describe "getJarPathFromPom" $ do
+    it' "should return False when jar does not exist" $ do
+      jarFile <- sendIO missingJar
+      result <- isValidJar jarFile
```

Comment:
Should we have tests for `isValidJar` and your other exported testing functions?